### DISCUSSION TITLE
Not staying signed in

### DISCUSSION THREAD

**Discussion by DracoClaw** [community/community#15639] [Copilot Conversations]
Every time I open VSC, a login popup for GitHub shows up. I've determined this is coming from Copilot, but would like help stopping the issue.

**Comment by PythonLimited** [comment]
Same here - also reinstalled VSC without luck.

Edit:
Downgraded to v1.17.5850 - no issues so far and doesn't require a re-login every time?
Maybe there was a recent change that caused this...
